### PR TITLE
Button & Input docs (#864)

### PR DESCRIPTION
**Pull request by iRoachie**
* Button & Input docs

* Updated Button with new props

### CODE REVIEW

File: docs/API/buttons.md [hunk 1]
```diff
@@ -1,69 +1,74 @@
-Buttons take a title and an optional [material icon name](https://design.google.com/icons/), as well as the props below.
+Buttons can be used to interact with the screen. It takes the following props.
 
-> You can override Material icons with one of the following: material-community, simple-line-icon, zocial, font-awesome, octicon, ionicon, foundation, evilicon, or entypo by providing an icon.type as a prop.
-
-![Buttons](http://i.imgur.com/CVf1xbr.png)
+![Buttons](https://i.imgur.com/ptrwAVW.png)
```
Comment (Monte9) [inline docs/API/buttons.md]: this image is huge... Let's add a width to it.

File: docs/API/input.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+![Input](https://i.imgur.com/Zm7MTQD.png)
```
Comment (Monte9) [inline docs/API/input.md]: let's add a width to this image too

File: docs/API/input.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  shake={true}
+/>
+
+<Input
+  placeholder='INPUT WITH ERROR MESSAGE'
+  displayError={true}
+  errorStyle={{ color: 'red' }}
+  errorMessage='ENTER A VALID ERROR HERE'
+/>
+
```
Comment (Monte9) [inline docs/API/input.md]: we should add 1-2 more complex examples here. Look at the login screens on the example app and copy and paste an entire input component with error validation, keyboard type containerStyle etc.

Ping me if you don't know what i am talking about.

File: docs/API/input.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { Input } from 'react-native-elements';
+
+<Input
+  placeholder='BASIC INPUT'
+/>
+
+<Input
+  placeholder='INPUT WITH ICON'
+  icon={
```
Comment (xavier-villelegier) [inline docs/API/input.md]: `icon` props was updated with the SearchBar PR and is now `leftIcon` and `rightIcon`

File: docs/API/input.md (hunk 3 above)
Comment (iRoachie) [reply docs/API/input.md]: Understand you perfectly. Going to tackle this in a future update to docs cause I wan't to change the layout without how we have examples and the preview of them

**Pull request merged**